### DISCUSSION TITLE
Get message from mq “mq-req” and reply with resp 00 then put “mq-resp”

### DISCUSSION THREAD

**Discussion by mehrdad2000** [tilln/jmeter-iso8583#101] [General]
Hi
Want to get message from mq queue name “mq-req” and reply with resp 00 in field 39 then put to “mq-resp” queue.
Is it possible to do this with jmeter-iso8583?

Thanks

**Comment by tilln** [comment]
Sorry, I don't understand the question or problem here.

**Reply by mehrdad2000** [reply]
@tilln here is the scenario:
jmeter1 > mq > jmeter2

Jmeter1 generate request 0100 and put into queue that call “q-req” 
Jmeter2 get request from queue “q-req” 
Jmeter2 set DE39=00 put response in queue that call “q-resp” 
Jmeter1 get response from “q-resp”

**Reply by tilln** [reply]
Ok, sure (though it seems strange that there is a JMeter instance on both ends and no test system anywhere).

I think you should be able to use your JMSChannel for this, assuming it's working properly.

So still not clear what the actual problem is.
